nilearn/decoding/tests/test_decoder.py

```diff
@@ -626,8 +734,8 @@ def test_decoder_apply_mask():
     assert model.masker_.smoothing_fwhm == smoothing_fwhm
 
 
-def test_decoder_split_cv():
-    X, y, _ = _make_multiclass_classification_test_data()
+def test_decoder_multiclass_classification_split_cv(multiclass_data):
```

Comment:
This test is verifying that errors and warnings are properly raised which is not obvious from the function name. Besides, there is one call to `pytest.raises()` which could be turned into a context manager for consistency.
Also, I think we could split in three different tests since we are checking three different things. WDYT ?

Reply:
I at least separated errors and warnings into different tests and used a context manager.